pages/about.html

```diff
@@ -4,9 +4,64 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me!</title>
+  <link href="../styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+  <header class="social-links">
+    <a href="https://www.instagram.com/jessica.w.roberts/">
+    <img src="../images/Instagram.png" alt="Instagram Logo" height=25 width=25>
+    </a>
+    <a href="https://www.linkedin.com/in/jessicawroberts/">
+      <img src="../images/Linkedin.png" alt="Linkedin Logo" height=25 width=25>
+      </a>
+  </header>
+  <nav>
+    <ul class="main-nav">
+      <li><a href="index.html">home</a></li>
+      <li class="hash">/</li>
+      <li><a href="about.html">about</a></li>
+      <li class="hash">/</li>
+      <li><a href="portfolio.html">portfolio</a></li>
+      <li class="hash">/</li>
+      <li><a href="musings.html">musings</a></li>
```

Comment:
You should use CSS to make these slashes instead of adding empty list items:

```suggestion
      <li><a href="index.html">home</a></li>
      <li><a href="about.html">about</a></li>
      <li><a href="portfolio.html">portfolio</a></li>
      <li><a href="musings.html">musings</a></li>
```

You can use something like this:

```css
.main-nav li::after {
    content: "/";
}

.main-nav li::after:last-child {
    display: none;
}
```

See: https://developer.mozilla.org/en-US/docs/Web/CSS/::after